backup/bin/setup-barman

```diff
@@ -109,21 +106,15 @@ def initial_backup(conf):
     subprocess.run(["barman", "cron"], check=True)
     for x in conf:
         nm = x['configuration_name']
-        force_archive(nm)
-        print("Running initial backup of {}".format(nm), flush=True)
-        subprocess.run(["barman", "backup", nm], check=True)
-        print("Checking barman status of {}".format(nm), flush=True)
-        subprocess.run(["barman", "check", nm], check=True)
-
-
-def setup_cron(conf):
```

Comment:
So we no longer need this because the schedule.yaml file for yacron handles this now.

Reply:
yes, that's right